packages/api/src/Api.ts

```diff
@@ -67,13 +65,13 @@ export class Api extends ApiPromise {
     options.metadata = Object.assign(staticMetadata, options.metadata);
     options.types = { ...options.types, ...Types };
     options.derives = mergeDeriveOptions(derives, options.derives);
-    options.rpc = { ...(rpc as any), ...options.rpc };
+    options.rpc = { ...rpc, ...options.rpc };
 
     super(options as ApiOptionsBase);
   }
 }
 
-async function withTimeout(promise: Promise<Api>, timeoutMs: number = DEFAULT_TIMEOUT): Promise<Api> {
+async function withTimeout(promise: Promise<Api>, timeoutMs: number): Promise<Api> {
```

Comment:
what is the new default timeout / behaviour here?

Reply:
@alexsednz ?

Reply:
The same as before. DEFAULT_TIMEOUT is moved to the same file where getTimeout is implemented. So instead of having DEFAULT_TIMEOUT in Api and then importing it in several places including ApiRx (which is considered a sibling to Api, meaning at the same level in the hierarchy) Now we call getTimeout and getTimeout will return the user-specified timeout or otherwise the default value.